src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/help-gallerySubscriptionId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The subscription id of the target gallery image belonged to. Leave empty if it is in the same subscription as the Azure Service Principal.
```

Comment:
service principal doesn't has subscription_id, but role assigned to some subscription. how about "same subscription" as configured at  xxxxx

Reply:
Updated